bin/node/cli/src/service.rs

```diff
@@ -161,6 +171,27 @@ fn set_prometheus_registry(config: &mut Configuration) -> Result<(), ServiceErro
 	Ok(())
 }
 
+pub fn open_frontier_backend(config: &Configuration) -> Result<Arc<dc_db::Backend<Block>>, String> {
+	let config_dir = config
+		.base_path
+		.as_ref()
+		.map(|base_path| base_path.config_dir(config.chain_spec.id()))
+		.unwrap_or_else(|| {
+			BasePath::from_project("", "", &crate::cli::Cli::executable_name())
```

Comment:
Check and test data dir

Reply:
Based on the current implementation:
```rs
let database_dir = config_dir.join("frontier").join("db");
```
the backend db path is `/data/alice/chains/pangolin/frontier/db`， maybe we should change it to `/data/alice/chains/pangolin/dvm/db`.  I need your advice at this point. @HackFisher

Reply:
Yes, let's change it to `config_dir.join("dvm").join("db");`